beacon-chain/types/block.go

```diff
@@ -271,6 +272,21 @@ func (b *Block) isAttestationValid(attestationIndex int, db beaconDB, aState *Ac
 		return false
 	}
 
+	// Retrieve block from db using justified_block_hash and
+	// verify its slot number is the same as attestation justified slot number.
+	rBlock, err := db.GetBlock(hash)
+	if err != nil {
+		log.Errorf("Failed to get block: %v", err)
+		return false
+	}
+	if attestation.JustifiedSlot != rBlock.SlotNumber() {
+		log.Errorf("The attestation's slot number is not the same as the slot number of the block retrieved by the justified slot hash. " +
+			"Attestation slot number: %d",
+			attestation.JustifiedSlot)
+		log.Errorf("Retrieved block slot number: %d",rBlock.SlotNumber())
```

Comment:
Please make this part of the error message on the previous line. This is one issue and only one error should be logged

Reply:
I think I resigned the CLA. If it did not work, please let me know and I will try again.